### PR TITLE
chore: Improve errors in `sv::messages` attribute (#345)

### LINKED ISSUE #345: Test `sv::messages` errors

### PR DESCRIPTION
**Pull request by jawoznia**
closes #345

### CODE REVIEW

File: sylvia/tests/ui/attributes/messages/invalid_custom.rs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+mod interface {
+    use sylvia::cw_std::StdError;
+
+    #[sylvia::interface]
+    pub trait Interface {
+        type Error;
+    }
+
+    impl Interface for crate::Contract {
```
Comment (kulikthebird) [inline sylvia/tests/ui/attributes/messages/invalid_custom.rs:13]: nit: Let's move the interface impl to the contract's module

File: sylvia/tests/ui/attributes/messages/unexpected_token.rs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    use sylvia::cw_std::{Empty, StdError};
+    use sylvia::types::CustomMsg;
+
+    #[sylvia::interface]
+    pub trait Interface {
+        type Error;
+        type ParamT: CustomMsg;
+    }
+
+    impl Interface for crate::Contract {
```
Comment (kulikthebird) [inline sylvia/tests/ui/attributes/messages/unexpected_token.rs:15]: Ditto

**Review by kulikthebird — APPROVED** [review]
3 optional comments, lgtm

File: sylvia/tests/ui/attributes/messages/invalid_custom.rs (hunk 1 above)
Comment (jawoznia) [reply sylvia/tests/ui/attributes/messages/invalid_custom.rs:13]: This is simplified as this is not supposed to be an example.

**Pull request merged**